### ISSUE TITLE
Feedback of milestone 2

### ISSUE THREAD

**Issue by arthursunbao** [UBC-MDS/ezmodel#20]
Hi All,

Nice work on milestone 2 and here is my comment :

1. For your first line of your python file, it is suggested that you can include #!/usr/bin/env python just in case the user is running your code in Linux(like me)
2. Good documentation of comments in ezmodel.py with clear explanation for functions for input and output.
3. ezmodel.py: commented lines from 137 to 149 should not be in the published source code
4. ezmodel.py: Suggest to refer to https://google.github.io/styleguide/pyguide.html for Python coding style improvement including how many spaces between each line 
5. ezmodel.py: class Socre can be divided into another .py file
6. ezmodel.py: line 235： np.random.randint(1, 99999999)， why 9999999?
7. ezmodel.py: I like your handling the data structure in line 257
8. ezmodel.py: I like the class you wrote for Score, with good manipulation of python syntax sugar, but pay attention to styling(indentation and space between lines)

Regards
Jason

**Comment by tylercroberts** [comment]
Commit addressing comments 1, 5, and 8: aa64a4ca50c6fbd3e7e7154657c087c8e87e3e54

For comment 3, did you mean 37 to 49? I cannot find commented out code in the sections you mentioned for the release you'd be commenting on.

As for 6, 99999999 was selected simply because it's a large number. I suppose `Integer.MAX_VALUE` might be better. Would you suggest we change it?

**Comment by arthursunbao** [comment]
Yes. I would recommend you to change that since Interger.MAX_VALUE make
sense.



2018-03-16 13:00 GMT-07:00 Tyler Roberts <notifications@github.com>:
